### DISCUSSION TITLE
Idea: allow to have a wav as reference for editing midi

### DISCUSSION THREAD

**Discussion by ArielAram** [ryohey/signal#273] [Ideas]
Hi!
An idea, based on a real need.

There are many mp3 to midi converters around. The most recent is basic pitch by Spotify. Akoff Media labs and Anthemscore are other converters. All converters produce midi files with errors (wrong detections, etc).

What I miss is a efficient, intuitive, easy to use tool to edit the midi by comparing to the audio: where you select a fragment and listen to the fragment in the wav, the midi, and then make corrections to the midi (delete a note, or change its duration or starting point, etc). Such a tool does not require advanced algorythms....but it must be well designed, as you will spend hours cleaning a midi, any unnecessary extra click will become thousands of unnecessary clicks.

What do you think ?

regards
Ariel//

**Comment by ryohey** [comment]
@ArielAram Thanks for your suggestion! I often do that too, but with a DAW - try Cubase, Studio One, or some other DAW software. I would rather signal be a MIDI-specific tool than a half-baked DAW.
